test/camel-case.js

```diff
@@ -63,7 +63,7 @@ describe('unCamelCase', () => {
 	});
 	describe('symbols', () => {
 		symbols.forEach((symbol) => {
-			it(JSON.stringify(symbol), () => {
+			it(`${JSON.stringify(symbol)}`, () => {
```

Comment:
[IMO] Just wrapping it with quotes seems easier to read, but this is just my opinion.
(you can ignore this comment 👌 )

```suggestion
			it(`"${symbol}"`, () => {
```

Reply:
Seems reasonable to me, doesn't break the test!